src/componants/calculetor.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
```

Comment:
- [ ] Kindly remove these linters disabling here. Linters help us write the best possible code, let's maintain that.

  **HINT:** You are mostly receiving these errors because you are adding events to `li` tags. You can refactor your `li` tags to `button` tags and add the `type="button"` attribute. Like so:

  ```
  <button type="button" onClick={this.eventHandler} id="plus-minus" className="key">
    +/-
  </button>
  ```